src/react-apps/applications/shared/src/navigation/drawer/TabletDrawerMenu.tsx

```diff
@@ -0,0 +1,153 @@
+import Button from '@material-ui/core/Button';
+import Collapse from '@material-ui/core/Collapse';
+import Divider from '@material-ui/core/Divider';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { leftDrawerMenuSettings } from './drawerMenuSettings';
+import { mainMenuSettings } from './drawerMenuSettings';
+import { styles } from './tabletDrawerMenustyle';
+
+export interface ITabletDrawerMenuProps {
+  classes: any;
+  theme: any;
+  tabletDrawerOpen: boolean;
+  handleTabletDrawerMenu: () => void;
+}
+
+export interface ITabletDrawerMenuProps extends WithStyles<typeof styles> { }
+
+class TabletDrawerMenu extends React.Component<ITabletDrawerMenuProps, any> {
+  public state = {
+    open: false,
+    openSubMenus: [] as number[],
+    selectedMenuItem: '',
+  };
+
+  public handleDrawerOpen = () => {
+    this.props.handleTabletDrawerMenu();
+  }
+
+  public handleDrawerClose = () => {
+    this.setState({ open: false });
+  }
+
+  public handleMenuItemClicked = (menuItem: any, id: number) => {
+    this.setState((state: any) => {
+      return {
+        selectedMenuItem: menuItem.displayText,
+      };
+    });
+    this.handleSubmenuClicked(id);
+  }
+
+  public handleSubmenuClicked = (id: number) => {
+    const openIdIndex = this.state.openSubMenus.indexOf(id);
+    this.setState((state: any) => {
+      if (openIdIndex > -1) {
+        state.openSubMenus.splice(openIdIndex, 1);
+      } else {
+        state.openSubMenus.push(id);
+      }
+      return {
+        openSubMenus: state.openSubMenus,
+      };
+    });
+  }
+
+  public render() {
+    const { classes } = this.props;
+
+    return (
+      <div>
+        <Button
+          size='small'
+          variant='outlined'
+          className={classes.button}
+          onClick={this.handleDrawerOpen}
+        >
+          {this.props.tabletDrawerOpen ? 'lukk' : 'meny'}
```

Comment:
Should use language api for texts.

Reply:
This will be handled in a separate issue